rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -194,12 +268,66 @@ s32 cellCameraOpen()
 s32 cellCameraOpenEx(s32 dev_num, vm::ptr<CellCameraInfoEx> info)
 {
 	UNIMPLEMENTED_FUNC(cellCamera);
```

Comment:
Since these functions now have some args and initial functionality, most of these `UNIMPLEMENTED_FUNC`s that are still left can be changed to actual log with args